barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -199,6 +200,22 @@ void writeVk(const std::string& bytecodePath, const std::string& outputPath)
     }
 }
 
+void writePk(const std::string& bytecodePath, const std::string& outputPath)
+{
+    auto constraint_system = get_constraint_system(bytecodePath);
```

Comment:
not a big deal but vscode has a refactoring thing to turn auto into the explicit type, can make the C++ easier to follow (compared to typescript, C++ tooling is rougher, so prefer more explicit types)